lib/tasks/webpacker/verify_install.rake

```diff
@@ -3,14 +3,16 @@ require "webpacker/configuration"
 namespace :webpacker do
   desc "Verifies if webpacker is installed"
   task verify_install: [:check_node, :check_yarn, :check_binstubs] do
-    if Webpacker.config.config_path.exist?
-      $stdout.puts "Webpacker is installed 🎉 🍰"
-      $stdout.puts "Using #{Webpacker.config.config_path} file for setting up webpack paths"
-    else
-      $stderr.puts "Configuration config/webpacker.yml file not found. \n"\
+    unless Webpacker.config.custom_compile?
```

Comment:
Now that we've extracted all the logic from the rake tasks into the Webpacker classes, I think the better move here, rather than trying to hack the webpacker tasks, is probably just to have your own reactonrails:* tasks that call these Webpacker classes directly. Then you can tailor output and prerequisites as you please 👍